source/NVDAObjects/UIA/edge.py

```diff
@@ -156,10 +161,9 @@ def _getControlFieldForObject(self,obj,isEmbedded=False,startOfNode=False,endOfN
 			field['placeholder']=obj.placeholder
 		# For certain controls, if ARIA overrides the label, then force the field's content (value) to the label
 		# Later processing in Edge's getTextWithFields will remove descendant content from fields with a content attribute.
-		ariaProperties=obj._getUIACacheablePropertyValue(UIAHandler.UIA_AriaPropertiesPropertyId)
-		hasAriaLabel=('label=' in ariaProperties)
-		hasAriaLabelledby=('labelledby=' in ariaProperties)
-		if field.get('nameIsContent'):
+		hasAriaLabel = 'label' in ariaProperties
+		hasAriaLabelledby = 'labelledby' in ariaProperties
+		if field.get('name	IsContent'):

```

Comment:
This looks like a mistake, there is whitespace between `name` and `IsContent`

I'm not currently reviewing this, was just trying to get an overview of how this will be impacted by #10371

Reply:
Thanks for catching this!